cl/audio/feeds.py

```diff
@@ -25,35 +32,56 @@ class JurisdictionPodcast(JurisdictionFeed):
     item_enclosure_mime_type = "audio/mpeg"
 
     def title(self, obj):
-        return f"Oral Arguments for the {obj.full_name}"
+        court = obj[1]
+        return f"Oral Arguments for the {court.full_name}"
+
+    def get_object(self, request, court):
+        return request, get_object_or_404(Court, pk=court)
 
     def items(self, obj):
         """
         Returns a list of items to publish in this feed.
         """
-        with Session() as session:
-            solr = ExtraSolrInterface(
-                settings.SOLR_AUDIO_URL, http_connection=session, mode="r"
-            )
-            params = {
+        request = obj[0]
+        court = obj[1]
+        if waffle.flag_is_active(request, "oa-es-deactivate"):
+            with Session() as session:
+                solr = ExtraSolrInterface(
+                    settings.SOLR_AUDIO_URL, http_connection=session, mode="r"
+                )
+                params = {
+                    "q": "*",
+                    "fq": f"court_exact:{court.pk}",
+                    "sort": "dateArgued desc",
+                    "rows": "20",
+                    "start": "0",
+                    "caller": "JurisdictionPodcast",
+                }
+                items = solr.query().add_extra(**params).execute()
+                return items
+        else:
+            cd = {
                 "q": "*",
-                "fq": f"court_exact:{obj.pk}",
-                "sort": "dateArgued desc",
-                "rows": "20",
-                "start": "0",
-                "caller": "JurisdictionPodcast",
+                "court": court.pk,
+                "order_by": "dateArgued desc",
+                "type": SEARCH_TYPES.ORAL_ARGUMENT,
             }
-            items = solr.query().add_extra(**params).execute()
+            search_query = AudioDocument.search()
+            items = do_es_podcast_query(search_query, cd, rows=20)
             return items
 
     def feed_extra_kwargs(self, obj):
+        if isinstance(obj, tuple):
+            court = obj[1]
+        else:
+            court = obj
```

Comment:
I don't think we need this if statement, the `obj` argument is always going to be a `tuple` because it will get its value from the `get_object` method.

Reply:
Yeah, as we talked about this in slack, obj can be a tuple since this method is also used by `AllJurisdictionsPodcast` and `SearchPodcast`. I simplified this part of the code by checking `isinstance(obj, tuple)` directly in the 'if' statement below.

Thanks!